dotcom-rendering/src/web/server/frontTemplate.ts

```diff
@@ -0,0 +1,248 @@
+                        }
+                    }
+
+                    // We've got contracts to abide by with the Ophan tracker
+                    // Setting pageViewId here ensures we're not getting race-conditions at all
+                    window.guardian.config.ophan = {
+                        // This is duplicated from
+                        // https://github.com/guardian/ophan/blob/master/tracker-js/assets/coffee/ophan/transmit.coffee
+                        // Please do not change this without talking to the Ophan project first.
+                        pageViewId:
+                            new Date().getTime().toString(36) +
+                            'xxxxxxxxxxxx'.replace(/x/g, function() {
+                                return Math.floor(Math.random() * 36).toString(36);
+                            }),
+                    };
+                </script>
+
+                <script>
+                    // Set the browserId from the bwid cookie on the ophan object created above
+                    // This will need to be replaced later with an async request to an endpoint
+                    (function (window, document) {
+
+                        function getCookieValue(name) {
+                            var nameEq = name + "=",
+                                cookies = document.cookie.split(';'),
+                                value = null;
+                            cookies.forEach(function (cookie) {
+                                while (cookie.charAt(0) === ' ') {
+                                    cookie = cookie.substring(1, cookie.length);
+                                }
+                                if (cookie.indexOf(nameEq) === 0) {
+                                    value = cookie.substring(nameEq.length, cookie.length);
+                                }
+                            });
+                            return value;
+                        }
+
+                        window.guardian.config.ophan.browserId = getCookieValue("bwid");
+
+                    })(window, document);
+                </script>
+
+				<script>
+					window.curlConfig = {
+						baseUrl: '${ASSET_ORIGIN}assets',
+						apiName: 'require'
+					};
+					window.curl = window.curlConfig;
+				</script>
+
+
+                <noscript>
+                    <img src="https://sb.scorecardresearch.com/p?c1=2&c2=6035250&cv=2.0&cj=1&cs_ucfr=0&comscorekw=${encodeURIComponent(
+						keywords,
+					).replace(/%20/g, '+')}" />
+                </noscript>
+                ${priorityScriptTags.join('\n')}
+                <style class="webfont">${getFontsCss()}</style>
+                <style>${resets.resetCSS}</style>
+				${css}
+				<link rel="stylesheet" media="print" href="${ASSET_ORIGIN}static/frontend/css/print.css">
+			</head>
+
+			<body>
+                ${html}
+                ${[...lowPriorityScriptTags].join('\n')}
```

Comment:
Why are we spreading this array here?

Reply:
This is copied code so is probably outside the scope of this PR but I think perhaps it ended up this way for legacy reasons? That is, maybe an earlier version had more than one item in the array?

Reply:
If it wants tech-debt eradicating probably worth merging this and change it.